src/components/Button/index.js

```diff
@@ -254,47 +258,44 @@ class Button extends Component {
                 onPressOut={this.props.onPressOut}
                 onMouseDown={this.props.onMouseDown}
                 disabled={this.props.isLoading || this.props.isDisabled}
-                style={[
+                wrapperStyle={[
                     this.props.isDisabled ? {...styles.cursorDisabled, ...styles.noSelect} : {},
                     styles.buttonContainer,
                     this.props.shouldRemoveRightBorderRadius ? styles.noRightBorderRadius : undefined,
                     this.props.shouldRemoveLeftBorderRadius ? styles.noLeftBorderRadius : undefined,
                     ...StyleUtils.parseStyleAsArray(this.props.style),
                 ]}
+                style={[
+                    styles.button,
+                    this.props.small ? styles.buttonSmall : undefined,
+                    this.props.medium ? styles.buttonMedium : undefined,
+                    this.props.large ? styles.buttonLarge : undefined,
+                    this.props.success ? styles.buttonSuccess : undefined,
+                    this.props.danger ? styles.buttonDanger : undefined,
+                    (this.props.isDisabled && (this.props.success || this.props.danger)) ? styles.buttonOpacityDisabled : undefined,
+                    (this.props.isDisabled && !this.props.danger && !this.props.success) ? styles.buttonDisabled : undefined,
+                    this.props.shouldRemoveRightBorderRadius ? styles.noRightBorderRadius : undefined,
+                    this.props.shouldRemoveLeftBorderRadius ? styles.noLeftBorderRadius : undefined,
+                    ...this.props.innerStyles,
+                ]}
+                hoverStyle={[
+                    (this.props.success && !this.props.isDisabled) ? styles.buttonSuccessHovered : undefined,
+                    (this.props.danger && !this.props.isDisabled) ? styles.buttonDangerHovered : undefined,
+                ]}
                 nativeID={this.props.nativeID}
+                accessibilityLabel={this.props.accessibilityLabel}
+                hoverDimmingValue={1}
```

Comment:
for this one too, could you help why did we need to set it as 1?

Reply:
same comment as above